web-client/src/views/DocketRecord/DocketRecord.tsx

```diff
@@ -287,3 +377,54 @@ export const DocketRecord = connect(
 );
 
 DocketRecord.displayName = 'DocketRecord';
+
+export function SortableHeader({
```

Comment:
I feel like exporting `SortableHeader` as a thin wrapper around `SortableColumn` might be a little confusing: e.g.,"Header" vs. "Column" even though they are the "same" under the hood, `SortableHeader` in principle has little to do with DocketRecord, etc. I wonder if it is worth passing in `hideOnMobile` straight into `SortableColumn` (with default = false) and maybe even renaming it (reflecting your new, better name) to `SortableColumnHeader`?